libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -633,6 +666,48 @@ export class ActivityHandler extends ActivityHandlerBase {
         await this.handle(context, 'UnrecognizedActivityType', this.defaultNextEvent(context));
     }
 
+    private getAdaptiveCardInvokeValue(activity: Activity): AdaptiveCardInvokeValue {
+        const { value } = activity;
+        if (!value) {
+            throw this.createAdaptiveCardInvokeException(
+                StatusCodes.BAD_REQUEST,
+                'BadRequest',
+                'Missing value property'
```

Comment:
This is a pretty benign error message, @johnataylor is this okay to send on the wire? Or would you rather this be logged somehow?

Reply:
A faithful port of https://github.com/microsoft/botbuilder-dotnet/pull/5227/files#diff-7a382816f3c128c2daeb75ea350f34c5410ff57febaa968ada7e17dabb039224R667